### DISCUSSION TITLE
The reset operation fires any event?

### DISCUSSION THREAD

**Discussion by cottonspace** [fengyuanchen/viewerjs#569] [Q&A] [answered]
Hello.

My application perform some procedures when "rotated" or "scaled" events fired.
But "reset" operation does not fire the "rotated" or "scaled" events.
How can I detect "reset" event?

**Comment by fengyuanchen** [comment, marked as answer]
Not support that currently. You may use the native [`transtionend`](https://developer.mozilla.org/en-US/docs/Web/API/Element/transitionend_event) event.

**Reply by cottonspace** [reply]
I appreciate your reply. I would try using the `transitionend` event.

Regards,
